docs/UserGuide.md

```diff
@@ -295,21 +310,41 @@ You are required to input the absolute path to the photo for this command. If yo
 
 ###  List all contacts: `list`
 
-Shows a list of all contacts.
+After using CampusConnect's [find contact](#find-contacts) feature that filters the contacts you are looking for, you may want to have an oversight of all your contacts again! Thus, CampusConnect allows you to list all your contacts.
```

Comment:
```suggestion
After using CampusConnect's [find contact](#find-contacts) feature that filters the contacts you are looking for, you may want to have an overview of all your contacts again! Thus, CampusConnect allows you to list all your contacts.
```
I think `oversight` is the wrong word choice here

Reply:
fixed in b33ffa82